rastreiobot.py

```diff
@@ -470,18 +470,29 @@ def cmd_magic(message):
          return 0
     log_text(message.chat.id, message.message_id, message.text)
     user = str(message.chat.id)
-    code = (
-        str(message.text.strip().replace('/start ', '').replace('\n', ' ')
+    message_text = (
+        message.text.strip().replace('/start ', '').replace('\n', ' ')
         .replace('/', '').upper().replace('@RASTREIOBOT', '')
-        .replace('📮 ', '').replace('📮', '').split(' ')[0])
+        .replace('📮 ', '').replace('📮', '').split(' ')
```

Comment:
Por que tirou o `[0]`?
A idéia é que a pessoa ao encaminhar uma mensagem do próprio bot ele retire todo o excesso da mensagem, deixando somente o código..
Ao retirar o `[0]` retornaria um array, não?

Reply:
Ah sim.. Acho que entendi.. Quem vai definir o que é `code` é ali no for..

Reply:
Exatamente. E caso não seja encontrado nenhum código de rastreio o code segue None como na logica anterior